### PR TITLE
feat(pair): add pair templates and graphics

### PR DESCRIPTION
**Pull request by vladikoff**
This can be reviewed with https://www.lucidchart.com/documents/view/9a420c19-1e92-42a5-8eae-908a442c1044/0

Extract from https://github.com/mozilla/fxa-content-server/pull/6876/files

### CODE REVIEW

File: app/scripts/templates/pair/index.mustache [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<div id="main-content" class="card">
+  <header>
+    <h1 id="pair-auth-complete-header">{{#t}}Pair using an app{{/t}}</h1>
```
Comment (vladikoff) [inline app/scripts/templates/pair/index.mustache]: https://www.lucidchart.com/documents/view/9a420c19-1e92-42a5-8eae-908a442c1044/6

File: app/scripts/templates/pair/failure.mustache [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<div id="main-content" class="card">
+  <header>
+    <h1 id="fxa-pair-failure-header">
+      {{#t}}Pairing not successful{{/t}}
```
Comment (vladikoff) [inline app/scripts/templates/pair/failure.mustache]: https://www.lucidchart.com/documents/view/9a420c19-1e92-42a5-8eae-908a442c1044/5

File: app/scripts/templates/pair/auth_wait_for_supp.mustache [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<div id="main-content" class="card pair-auth">
+  <header>
+    <h1 id="fxa-pair-auth-wait-for-supp-header">
+      {{#unsafeTranslate}}Approve this connection <small>from this device</small>{{/unsafeTranslate}}
```
Comment (vladikoff) [inline app/scripts/templates/pair/auth_wait_for_supp.mustache]: https://www.lucidchart.com/documents/view/9a420c19-1e92-42a5-8eae-908a442c1044/4

File: app/scripts/templates/pair/supp_wait_for_auth.mustache [hunk 4]
```diff
@@ -0,0 +1,14 @@
+<div id="main-content" class="card pair-auth">
+  <header>
+    <h1 id="fxa-pair-supp-wait-for-auth-header">
+      {{#unsafeTranslate}}Approve this connection <small>from this device</small>{{/unsafeTranslate}}
```
Comment (vladikoff) [inline app/scripts/templates/pair/supp_wait_for_auth.mustache]: https://www.lucidchart.com/documents/view/9a420c19-1e92-42a5-8eae-908a442c1044/4

File: app/scripts/templates/pair/auth_complete.mustache [hunk 5]
```diff
@@ -0,0 +1,20 @@
+<div id="main-content" class="card pair-auth pair-auth-complete">
+  <header>
+    <h1 id="pair-auth-complete-header">{{#t}}Device connected{{/t}}</h1>
```
Comment (vladikoff) [inline app/scripts/templates/pair/auth_complete.mustache]: https://www.lucidchart.com/documents/view/9a420c19-1e92-42a5-8eae-908a442c1044/7

File: app/scripts/templates/pair/success.mustache [hunk 6]
```diff
@@ -0,0 +1,15 @@
+<div id="main-content" class="card pair-auth pair-auth-complete">
+  <header>
+    <h1 id="pair-auth-complete-header">{{#t}}Device connected{{/t}}</h1>
```
Comment (vladikoff) [inline app/scripts/templates/pair/success.mustache]: https://www.lucidchart.com/documents/view/9a420c19-1e92-42a5-8eae-908a442c1044/7 

Simple success landing point, and r-b or fenix can choose to use this once redirected from native rust call.

File: app/scripts/templates/pair/supp_allow.mustache [hunk 7]
```diff
@@ -0,0 +1,24 @@
+<div id="main-content" class="card pair-auth">
+  <header>
+    <h1 id="fxa-pair-supp-allow-header">
+      {{#unsafeTranslate}}Approve connections <small>to begin syncing</small>{{/unsafeTranslate}}
```
Comment (shane-tomlinson) [inline app/scripts/templates/pair/supp_allow.mustache]: The text here is different to the [text below](https://github.com/mozilla/fxa-content-server/pull/6901/files#diff-f6cf8d9ddb30af4131b37fdc7dd65000R4):

```
Approve this connection <small>from this device</small>
```

Comment (vladikoff) [reply app/scripts/templates/pair/supp_allow.mustache]: yeap, `wait` vs `allow` text is different 👍

**Pull request merged**